elements/lisk-p2p/src/p2p.ts

```diff
@@ -541,9 +544,15 @@ export class P2P extends EventEmitter {
 	 * invoke an async RPC on Peers to give them our new node status.
 	 */
 	public applyNodeInfo(nodeInfo: P2PNodeInfo): void {
+		const validatedNodeInfo = validateNodeInfo(
```

Comment:
`validateNodeInfo` throws an error, and it's not being handled here. this function is being invoked in the [framework](https://github.com/LiskHQ/lisk-sdk/blob/19ffcf6b4fd4df95e1b7d39d2d64a3d0eb8ac421/framework/src/modules/network/network.js#L168) and it will result in error.